topics/statistics/tutorials/FNN/tutorial.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to feedforward neural networks (FNN)
+zenodo_link: https://zenodo.org/record/4660497
+questions:
+- What is a feedforward neural network (FNN)?
+- What are some applications of FNN?
+objectives:
+- Understand the inspiration for neural networks
+- Learn various activation functions, and classification/regression problems solved by neural networks
+- Discuss various cost/loss functions and the backpropagation algorithm
+- Learn how to create a neural network using Galaxy's deep learning tools
+- Solve a simple regression problem, car purchase price prediction, via FNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+---
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Artificial neural networks are a machine learning discipline roughly inspired by how neurons in a
+human brain work. In the past decade, there has been a huge resurgence of neural networks thanks
+to the vast availability of data and enormous increases in computing capacity (Successfully
+training complex neural networks in some domains requires lots of data and compute capacity). There
+are various types of neural networks (Feedforward, recurrent, etc). In this tutorial, we discuss
+feedforward neural networks (FNN), which have been successfully applied to pattern classification, 
+clustering, regression, association, optimization, control, and forecasting ({% cite JainEtAl %}. 
+We will discuss biological neurons that inspired artificial neural networks, review activation 
+functions, classification/regression problems solved by neural networks, and the backpropagation 
+learning algorithm. Finally, we construct a FNN to solve a regression problem using car purchase
+price prediction dataset. 
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Inspiration for artificial neural networks
+
+A neuron is a special biological cell with information processing ability ({% cite JainEtAl %}). 
+Figure 1 shows a biological neuron. It has a cell body and two outreaching tree-like branches: 
+axon and denderites. A neuron receives signals from other neurons through its denderites, and 
+transmits signals generated by its cell body to other neurons via its axon. A synapse is a place 
+of contact between two neurons, an axon strand of one neuron and a denderite strand of another 
+neuron. A synapse can either enhance or inhibit the signal that passes through it. Learning occurs 
+by changing the effectiveness of synapse. If the signals received exceeds a threshold, the neuron 
+*fires*, i.e. it transmits a signal to other neurons. If not, it will not fire.       
+
+![Sketch of a biological neuron and its components](../../images/FNN_bio_neuron.png "A biological neuron ({% cite JainEtAl %})")
```

Comment:
I'm not sure citations work in image captions, we should check this

Reply:
I just double checked locally. It works.

Reply:
fantastic! then it's fine from my side for the formatting.

merge when you're happy @anuprulez :tada: